rllib_contrib/a2c/BUILD

```diff
@@ -0,0 +1,40 @@
+# Learning Tests
+
+py_test(
+    name = "learning_tests_cartpole_a2c",
+    main = "run_regression_tests.py",
+    tags = ["team:rllib", "learning_tests", "learning_tests_cartpole", "learning_tests_discrete"],
+    size = "large",
+    srcs = ["tests/run_regression_tests.py"],
+    data = ["tuned_examples/cartpole-a2c.yaml"],
+    args = ["--dir=tuned_examples/"]
+)
+
+py_test(
+    name = "learning_tests_cartpole_a2c_microbatch",
+    main = "tests/run_regression_tests.py",
```

Comment:
can this be something like `rllib.tests.run_regression_tests`? we need to share this code between algorithms.